src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceContacts.java

```diff
@@ -0,0 +1,183 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.fundacionjala.sfdc.commons.DriverManager;
+
+/**
+ * SalesForceContacts.java
+ * Class that represents Contacts page in SalesForce.
+ */
+public class SalesForceContacts {
+    private WebDriver webDriver;
+    private By contactNameTextField;
+    private By contactLastNameTextField;
+    private By newContactButton;
+    private By saveContactButton;
+    private By editContactButton;
+    private By deleteContactButton;
+    private WebDriverWait wait;
+    private By newContactLabel;
+    private By lastContactLink;
+    private By homeContact;
+
+    /**
+     * Constructor.
+     * @param webDriver instance.
+     */
+    public SalesForceContacts(final WebDriver webDriver) {
+        this.webDriver = webDriver;
+        this.contactNameTextField = By.id("name_firstcon2");
+        this.saveContactButton = By.cssSelector("#topButtonRow> input[name='save']");
+        this.contactLastNameTextField = By.id("name_lastcon2");
+
+
+        this.editContactButton = By.cssSelector("#topButtonRow > input[name='edit']");
+        this.deleteContactButton = By.cssSelector("#topButtonRow > input[name='del']");
+        this.newContactLabel = By.className("topName");
+
+
+        this.newContactButton = By.cssSelector("input[name='new']");
+        this.homeContact = By.cssSelector("h2.pageDescription");
+        this.lastContactLink = By.cssSelector("tr.dataRow.even.first.dataRow >th[scope='row'] > a");
+    }
+
+    /**
+     * Method to get "edit" button.
+     * @return web element edit button.
+     */
+    public WebElement getEditButton() {
+        return this.webDriver.findElement(this.editContactButton);
+    }
+    /**
+     * Method to get "save" button.
+     * @return web element save button.
+     */
+    public WebElement getSaveContactButton() {
+        return this.webDriver.findElement(this.saveContactButton);
+    }
+    /**
+     * Method to know if an account is saved.
+     * @return the new contact name displayed in label.
+     */
+    public String newContactSavedName() {
+        return this.webDriver.findElement(this.newContactLabel).getText();
+    }
+    /**
+     * Method to know if an account was delete.
+     * @return the contact home page.
+     */
+    public String contactHomePage() {
+        wait = DriverManager.getInstance().getWaiter();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this instance-reference to a static reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2209)